### PR TITLE
[PH] Translation of 'en.md' files of 'rules' and 'osu!wiki_Contribution_Guide' directory

### PR DESCRIPTION
**Pull request by rhalp10**
### Description
Adding `ph.md ` Translation of _osu!wiki_Contribution_Guide/_ `en.md ` file and _rule/_ `en.md`

### Status

- [ ] WIP
- [x] Pending Review
- [ ] Finished

### Additional Notes 
Nothing

### CODE REVIEW

File: wiki/Rules/ph.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Ang isang hanay ng mga mas detalyadong mga tuntunin para sa konstruksiyon ng beatmap ay umiiral, na kilala bilang [Ranking Criteria](/wiki/Ranking_Criteria).
+
+1. **Tiyaking may pahintulot kang gamitin ang anumang nilalaman na kasangkot sa iyong beatmap.** TKabilang dito ang mga awit, video, mga tunog ng hit, graphics, at anumang iba pang nilalaman na hindi iyong sariling paglikha. Kung hindi ka sigurado kung saan makahanap ng nilalaman na libre upang magamit, mayroon kaming isang lumalagong library ng mga track na magagamit sa Listahan ng Mga [Tampok na Artist](https://osu.ppy.sh/beatmaps/artists/) ang lahat ng 100% na lisensyado para sa paggamit sa _osu!_.
+2.  **Manatili sa paksa sa mga talakayan ng Beatmap.** Ang mga thread / diskusyon ay tungkol sa beatmap na pinag-uusapan, at walang iba pa. Kung mayroon kang isang isyu sa isang bagay na hindi direktang tumutukoy sa beatmap, i-post ito sa naaangkop na forum.
+3.  **Huwag mag-plagiarise o magtangkang magnakaw ng gawain ng iba.** Huwag din mag-upload o gumamit ng trabaho ng ibang tao nang walang tahasang pahintulot (kabilang ang, ngunit hindi limitado sa, mga skin at mga kahirapan sa panauhin).
+
+Mga Panuntunan ng Off-Paksa
```
Comment (TPGPL) [inline wiki/Rules/ph.md]: you shouldnt translate the Off-Topic forum

File: wiki/osu!wiki_Contribution_Guide/ph.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Gabay sa Pag-ambag ng osu! wiki
+
+Maraming Salamat sa iyong interes sa paggawa ng osu! Wiki upang ito ay mapabuti! sa oras na ito, ang pagbibigay ng kontribusyon sa osu! Wiki ay nangangailangan ng mas maraming oras at bahagyang higit na pagsisikap upang maisagawa ang iyong mga pagbabago. Ang mabilisang gabay na ito ay inilaan para sa mga nagsisimula pa lamang at hindi pa gumagawa sa Github at ang osu!wiki workflow sa bago. Kung mayroon ka nang karanasan sa pag gamit ng GitHub, huwag mag-atubiling laktawan ang gabay na ito at sundin ang pangkaraniwang work-branch workflow para sa mga update na nilalaman ng imbakan na ito.
```
Comment (Joehuu) [inline wiki/osu!wiki_Contribution_Guide/ph.md]: `osu!wiki` -> `osu! wiki`
Can you do that for every instance? Including the name of the folder to `osu!_wiki_Contribution_Guide`. A recent PR was pushed with these minor changes.

**Comment by TPGPL** [comment]
can you apply review posts?

**Comment by rhalp10** [comment]
@TPGPL Sure, Sorry im a little bit busy.

File: wiki/osu!wiki_Contribution_Guide/ph.md (hunk 2 above)
Comment (rhalp10) [reply wiki/osu!wiki_Contribution_Guide/ph.md]: Sure

**Comment by Joehuu** [comment]
Close this because of inactivity.

**Comment by TPGPL** [comment]
Closing as there's no activity

**Pull request closed without merging**